news.html

```diff
@@ -5,11 +5,12 @@
 ---
 <ul class="news-posts">
   {% for post in site.posts %}
-  <li>
-    <a href="{{ post.url }}"><h2>{{ post.title }}</h2></a>
+  <li post-tags="{% for tag in post.tags %}{{ tag }} {% endfor %}" post-author="{{post.author}}">
+    <a href="{{ post.url }}"><h2 class="links">{{ post.title }}</h2></a>
     {% if post.subtitle %}
     <p>{{ post.subtitle }}</p>
     {% endif %}
-    <span>Published {{ post.date | date: "%d %b, %Y" }} by {{ post.author }}</span>
+    <span>Published {{ post.date | date: "%d %b, %Y" }} by <a class="links author" target="_blank" href="{{post.author-link}}">{{ post.author }}</a></span>
```

Comment:
Nice! Do you think it's worth not rendering the link if the author has no author-link?